src/components/TodoCard/TodoCard.jsx

```diff
@@ -8,12 +11,17 @@ import {
 } from '../../constants/constants';
 
 import PriorityIcon from '../PriorityIcon/PriorityIcon';
+import EditForm from '../EditForm/EditForm';
 import Menu from '../Menu/Menu';
 
 function TodoCard({
     todo,
-    onDelete
+    onDelete,
+    editTodo
 }) {
+
+    const [isEdit, setIsEdit] = useState(false);
```

Comment:
let's allow only one item for amendment. Now you can edit multiple.
In case if user tries to edit another task you should cancel all active amendments

<img width="635" alt="image" src="https://user-images.githubusercontent.com/35139442/208308700-13b937f9-f1df-463c-87be-d8c13a42d41f.png">